src/webui/www/public/scripts/contextmenu.js

```diff
@@ -284,6 +286,11 @@ var TorrentsTableContextMenu = new Class({
                 all_are_force_start = false;
             else
                 there_are_force_start = true;
+
+            if (data['auto_tmm'] === true)
+                there_are_auto_tmm = true;
+            else
+                all_are_auto_tmm = false;
```

Comment:
you can view `there_are_auto_tmm` as the logical disjunction of TMM properties of all torrents
and `all_are_auto_tmm` is the logical conjunction of them.
also its a lambda function which is looped through every torrents at line 262.